src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenShortCircuitOperatorTests.cs

```diff
@@ -4176,7 +4176,7 @@ static void Main()
             var comp = CompileAndVerify(source, expectedOutput: @"Success");
         }
 
-        [Fact(Skip = "836"), WorkItem(836, "GitHub")]
+        [Fact(), WorkItem(836, "GitHub")]
         public void ConditionalMemberAccessRace002()
```

Comment:
Consider removing it to `ConditionalMemberAccessRaceCondition002` (and also its siblings).